src/AgStateMachine.cpp

```diff
@@ -7,6 +7,12 @@
 
 #define SENSOR_CO2_CALIB_COUNTDOWN_MAX 5 /** sec */
 
+#define RGB_COLOR_R 255, 0, 0     /** Red */
```

Comment:
R -> RED, ....

Reply:
Both. Refactoring and fixing CO2 color for purple.